packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.js

```diff
@@ -98,7 +100,7 @@ export default class QuanticTab extends LightningElement {
 
   updateState() {
     this._isActive = this.tab?.state?.isActive;
-    this.shouldDisplay = this.searchStatus?.state?.firstSearchExecuted;
+    this.shouldDisplay = this.searchStatus?.state?.firstSearchExecuted && !this.hasInitializationError;
```

Comment:
Question: wouldn't updateState() not be called in case of initializationError? Why check for init error at this point?

Reply:
in the case of an initialization error updateState is not called no, good point! I should keep them separate.

Reply:
Just rechecked, the logic here is the following:
We should display the tab only when the first search has been executed AND there are no error during initialization.

I'm including this condition about the error initialization here to avoid having to add an additional if:false template in the HTML.
 instead of doing: 
```
<template if:true={hasInitializationError}>Error</template>
<template if:false={hasInitializationError}>
    <template if:true={shouldDisplay}>Tab component</template>
</template>
```

I'm updating the shouldDisplay condition to be `this.searchStatus?.state?.firstSearchExecuted && !this.hasInitializationError;`
 in order to have:

```
<template if:true={hasInitializationError}>Error</template>
<template if:true={shouldDisplay}>Tab component</template>
```
Both are equivalent, the second simpler imo.

Reply:
ah ok I see I get why the html file had no <template if:false={hasInitializationError}> then :-D